media/js/base/stub-attribution.js

```diff
@@ -336,8 +336,23 @@ if (typeof window.Mozilla === 'undefined') {
 
     StubAttribution.hasValidData = function(data) {
         if (typeof data.utm_content === 'string' && typeof data.referrer === 'string') {
+            var content = data.utm_content;
```

Comment:
I am wondering if we should restrict the length of the `utm_content` value given that we introduced a loop below given that we can add a lot of `25` after `rta%` in a string like `rta%25253AdUJsb2NrMEByYXltb25kaGlsbC5uZXQ`

Reply:
@willdurand what would you consider a sensible length for RTAMO referrals?

Reply:
Actually, we don't really need to be concerned with preserving the actual value here, since we're only validating. We can just check the first 10 or so characters

Reply:
> @willdurand what would you consider a sensible length for valid RTAMO referrals?

I'd say `120` for the length of the `content` value.